mne/html_templates/report/html.html.jinja

```diff
@@ -3,7 +3,7 @@
     <button class="accordion-button pt-1 pb-1" type="button" data-bs-toggle="collapse"
       data-bs-target="#accordion-collapse-{{ id }}" aria-expanded="true" aria-controls="accordion-collapse-{{id}}">
       <div class="w-100">
-        <span class="me-auto">{{ title }}</span>
+        <span class="me-auto"><a href="#{{ id }}" class="text-decoration-none">{{ title }}</a></span>
```

Comment:
FYI I also added these for links *in the main bar*. When you click them they don't go to that location, they still collapse the section -- but you can at least right-click-copy them. I think ideally these would live in a little anchor symbol that just set the URL at the top when you clicked it, but that wasn't trivial. This is at least some progress in the right direction I think.